.gitlab-ci.yml

```diff
@@ -518,6 +531,27 @@ ubuntu-canal-kubeadm-triggers:
   when: on_success
   only: ['triggers']
 
+coreos-weave-kubeadm-rbac:
+  stage: deploy-gce-part1
+  <<: *job
+  <<: *gce
+  variables:
+    <<: *gce_variables
+    <<: *coreos-weave_kubeadm_variables
```

Comment:
missing rbac vars?

Reply:
it's there on line 302. rbac is enabled by default in kubeadm anyway. We'll default all to rbac soon